Makefile

```diff
@@ -4,6 +4,8 @@ CURRENT_DIR := $(patsubst %/,%,$(dir $(realpath $(MKFILE_PATH))))
 
 # Ensure GOPATH
 GOPATH ?= $(HOME)/go
+# assume last entry in GOPATH is home to project
+GOPATH := $(lastword $(subst :, ,${GOPATH}))
```

Comment:
I wonder if this should be using the `go env` pattern:

https://github.com/hashicorp/consul/blob/ffcfdf29fc86d21a8be82ab5ccbaf52fea822171/GNUmakefile#L21

Not really a big deal just seems like it would be more accurate across platforms.

Reply:
Good call. I'll  update that.